### ISSUE TITLE
private hyperparameter search

### ISSUE THREAD

**Issue by dionman** [tensorflow/privacy#76]
is there any support for accounting privacy loss for hyperparameter search when training a model using dp sgd?

**Comment by npapernot** [comment]
We don't have a specific analysis tool to take into account the hyperparameter search at this time
